spec/struct.dd

```diff
@@ -34,6 +34,50 @@ $(GNAME AggregateBody):
     $(D {) $(GLINK2 module, DeclDefs)$(OPT) $(D })
 )
 
+    $(P The following example declares a struct type with a single integer field:)
+
+$(SPEC_RUNNABLE_EXAMPLE_RUN
+---
+struct S
+{
+    int i;
+}
+
+void main() {
+    S a;
+    a.i = 3;
+
+    S b = a; // copy a
+    a.i++;
+    assert(a.i == 4);
+    assert(b.i == 3);
+}
+---
+)
+
+    $(P A struct is allocated on the stack by default. To allocate on
```

Comment:
Only for local variables

Reply:
You can also say "struct instance". All local variables are stored on the stack but a class is a pointer to an instance

Reply:
Updated.